src/services/WolfDen.Domain/Entity/LeaveType.cs

```diff
@@ -16,13 +16,13 @@ public class LeaveType
         public int? DaysChekcMore { get; private set; }
         public int? DaysCheckEqualOrLess { get; private set; }
         public int? DutyDaysRequired { get; private set; }
-        public RestrictedLeaveType? RestrictionType { get; private set; }
-        public bool? Sandwich { get; private set; }
+        public bool? Sandwich { get; private set; }
+        public LeaveTypeEnum? Type { get; private set; }
```

Comment:
Inform mukil to change this field to Id and use  names other than type for eg: LeaveCategory for this enum